cabal.project

```diff
@@ -219,9 +219,8 @@ allow-newer:
   monoidal-containers:aeson,
   size-based:template-haskell
 
--- NOTE: On a branch to allow non-Typable DynLogicModels
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/quickcheck-dynamic
-  tag: 9659630c68c91b86f3007f30b9aa02675bf8c929
+  tag: c272906361471d684440f76c297e29ab760f6a1e
   --sha256: 1b9ppgavqad78a2z1zxv7v4jasjz6zz0mxkr0zx0bbcd0i00jajf
```

Comment:
The hash is now different, but it seems like our nix pipeline doesn't care :shrug: